### DISCUSSION TITLE
Error running Example 1 with Designsafe

### DISCUSSION THREAD

**Discussion by BofanYu** [NHERI-SimCenter/SimCenterCommon#219] [R2D]
Hi, 

I am using Example 1 as a try to use Designsafe. However, I got a error message in the the log file: "/work2/00477/tg457427/stampede3/SimCenterBackendApplications/v4.5.0/applications/createEVENT/SimCenterEvent/SimCenterEvent.py" "--filenameAIM" "/scratch/10000/allen777/tapis/ab149f8b-276c-41b4-82b0-4851a4fb2d23-007/tmp.SimCenter8a800eba-ddcf-4b74-8176-9bf4b1060212/Results/Buildings/8000-AIM.json" "--filenameEVENT" "EVENT.json" "--getRV" 

         -----------------------------------------------------------------------
21:49:46  ERROR
21:49:46 return code: 1

Could you please help me debug?

Allen,
Thank you

**Comment by fmckenna** [comment]
I made a change to the backend code on DesignSafe. it should run now. can you try again. thanks. frank

**Reply by BofanYu** [reply]
Thank you. The issue has been resolved.
